scripts/mainnet/reset.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash -x
+
+usage() {
+  cat << EOT
+Usage: $0 <<BUILD ID>>
+Reset fractals mainnet, example BUILD ID = e1ca4db_20211011_1204
+EOT
+}
+
+subsitute_node_build_id() {
+  ssh $1 "PERL_BADLANG=0 perl -pi -e \"s/\\\\w+_\\\\d+_\\\\d+/$2/\" docker-compose.yml"
+}
+
+# /p2p/12D3KooWB3ENNs5vK5HNJTHVd33wEZnXdoTFwHkEUcFdQ6JxRhRJ
+subsitute_bootnode_id() {
+  ssh $1 "PERL_BADLANG=0 perl -pi -e \"s|/p2p/\w+|/p2p/$2|\" docker-compose.yml"
+}
+
+(( $# < 1 )) && { echo "1 arguements is required."; usage; exit 1; }
+
+NODE_BUILD_ID=$1
+
+node_00="ubuntu@node-boot.mainnet.fractalprotocol.com"
```

Comment:
Since we treat this node differently, we should probably have the name different from the others.